api_EPA.py

```diff
@@ -57,6 +74,25 @@ def main():
 
     release_dataset()
 
+def get_start_time():
+
+    client = bigquery.Client()
+
+    start_time = """SELECT DATETIME_ADD(MAX(PARSE_DATETIME('%Y-%m-%dT%H:%M', UTC)), INTERVAL 1 hour) as next_dt 
+                    FROM redivis.epa_test.pm2_5:1"""
+
+    query_job = client.query(start_time)
+
+    print("Fetching EPA data")
```

Comment:
This print statement is a little confusing — we're fetching the last observation date in the previously uploaded data